clang/lib/AST/ByteCode/Interp.h

```diff
@@ -2474,9 +2476,9 @@ inline bool DoShift(InterpState &S, CodePtr OpPC, LT &LHS, RT &RHS) {
     if (!S.noteUndefinedBehavior())
       return false;
     RHS = -RHS;
-    return DoShift<LT, RT,
```

Comment:
Please dont' do unrelated changed in the same patch.   It looks like you perhaps tried to clang-format the whole document?  We only want to clang-format the code changes we make, which the `git clang-format HEAD^`  should do for you.